docs/getting-started.md

```diff
@@ -38,19 +38,14 @@ using Bzlmod. Here is a simplified setup to download the prebuilt runtimes.
 ```starlark
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-
-# Update the SHA and VERSION to the lastest version available here:
-# https://github.com/bazelbuild/rules_python/releases.
-
-SHA="84aec9e21cc56fbc7f1335035a71c850d1b9b5cc6ff497306f84cced9a769841"
-
-VERSION="0.23.1"
+# Update the snippet based on the latest release below
```

Comment:
nit: Are you sure about this change? I'm +1 on increasing to a newer version. Doesn't the extracted variables make it easier to bump versions in future? Or are you thinking there aren't major benefits to the indirection?

Reply:
I am thinking that one needs to copy the sha and version separately instead of just replacing the whole block of `http_archive`. If we had the indirection in the `releases` page, then sure, I think we should keep it, but the `releases` page does not have that.

What do you think?

Reply:
Oh right. Yes. If we could prepare a full copy-paste snippet somewhere that is kept up to date automatically, that would be cool. Im fine with your change.

Reply:
Yeah, the snippet is in the `releases` page, e.g. https://github.com/bazelbuild/rules_python/releases/tag/0.36.0